jenkins/ci.suse.de/cloud-mkphyscloud-qa-test-upgrade-ui.yaml

```diff
@@ -140,7 +142,7 @@
               onadmin_upgrade_ses_to_4
             fi
 
-            export cloudsource=$cloudsource
+            export cloudsource=$original_cloudsource
```

Comment:
The problem is I need the original cloudsource for cct later on until I get to the point where the admin node is already upgraded.